apps/server-monolith/app/models/pod.rb

```diff
@@ -58,25 +56,14 @@ def avatar_data
     }
   end
 
-  def pod_as_json
-    {
-      id: id,
-      avatar_data: avatar_data,
-      bio: bio,
-      domain: username,
-      name: display_name,
-    }
+  # Check if current pod is current user's personal pod
+  def personal
+    id === Current.user&.id
```

Comment:
Add `personal` and `owned` method to replace `*_as_json`, but dependent on `Current` model